node/sc/bridge_test.go

```diff
@@ -665,3 +665,313 @@ func TestExtendedBridgeAndCallbackERC721(t *testing.T) {
 		t.Fatalf("registerOfferEvent was not found.")
 	}
 }
+
+func generateBridgeTokenTestEnv(t *testing.T) (*backends.SimulatedBackend, *bind.TransactOpts, *bind.TransactOpts, *bridge.Bridge, *sctoken.ServiceChainToken, *scnft.ServiceChainNFT, common.Address, common.Address) {
+	key, _ := crypto.GenerateKey()
+	operator := bind.NewKeyedTransactor(key)
+	operator.GasLimit = gasLimit
+
+	testKey, _ := crypto.GenerateKey()
+	tester := bind.NewKeyedTransactor(testKey)
+	tester.GasLimit = gasLimit
+
+	alloc := blockchain.GenesisAlloc{operator.From: {Balance: big.NewInt(params.KLAY)}, tester.From: {Balance: big.NewInt(params.KLAY)}}
+	backend := backends.NewSimulatedBackend(alloc)
+
+	// Deploy Bridge
+	bridgeAddr, tx, b, err := bridge.DeployBridge(operator, backend, true)
+	assert.NoError(t, err)
+	backend.Commit()
+	assert.Nil(t, bind.CheckWaitMined(backend, tx))
+
+	// Deploy ERC20
+	erc20Addr, tx, erc20, err := sctoken.DeployServiceChainToken(operator, backend, bridgeAddr)
+	assert.NoError(t, err)
+	backend.Commit()
+	assert.Nil(t, bind.CheckWaitMined(backend, tx))
+
+	// Register ERC20
+	tx, err = b.RegisterToken(operator, erc20Addr, erc20Addr)
+	assert.NoError(t, err)
+	backend.Commit()
+	assert.Nil(t, bind.CheckWaitMined(backend, tx))
+
+	// Charge token to tester
+	tx, err = erc20.Transfer(operator, tester.From, big.NewInt(100))
+	assert.NoError(t, err)
+	backend.Commit()
+	assert.Nil(t, bind.CheckWaitMined(backend, tx))
+
+	// Deploy ERC721
+	erc721Addr, tx, erc721, err := scnft.DeployServiceChainNFT(operator, backend, bridgeAddr)
+	assert.NoError(t, err)
+	backend.Commit()
+	assert.Nil(t, bind.CheckWaitMined(backend, tx))
+
+	// Register ERC721
+	tx, err = b.RegisterToken(operator, erc721Addr, erc721Addr)
+	assert.NoError(t, err)
+	backend.Commit()
+	assert.Nil(t, bind.CheckWaitMined(backend, tx))
+
+	// Mint token to tester
+	tx, err = erc721.RegisterBulk(operator, tester.From, big.NewInt(0), big.NewInt(10))
+	assert.NoError(t, err)
+	backend.Commit()
+	assert.Nil(t, bind.CheckWaitMined(backend, tx))
+
+	return backend, operator, tester, b, erc20, erc721, erc20Addr, erc721Addr
+}
+
+// TestBridgeContract_InitStatus checks initial lock status.
+func TestBridgeContract_InitStatus(t *testing.T) {
+	_, _, _, b, _, _, erc20Addr, erc721Addr := generateBridgeTokenTestEnv(t)
+
+	// initial value check
+	isLocked, err := b.LockedTokens(nil, erc20Addr)
```

Comment:
How about changing the name to IsLockedTokens to match the name with IsLockedKLAY?

Reply:
There are many tokens. So I wrote. What do you think? :)